### PR TITLE
docs: elaborate on commitlint rules

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,17 +89,19 @@ the VCS/git logs more valuable. The general structure of a commit message is as
...
+    - `refactor`: modification of the code base which neither adds a feature nor fixes a bug - such as removing redundant code, simplifying code, renaming variables, etc.
     - `revert`: revert previous commits.
     - `test`: all changes related to tests such as refactoring existing tests or adding new tests.
     - `vim-patch`: all patches from upstream Vim. The commit messages for patches has [slightly different rules](https://github.com/neovim/neovim/wiki/Merging-patches-from-upstream-Vim#pull-requests) as not to interfere with existing scripts.
     - `chore`: Lastly, if none of the types above fits you may use `chore` as the type.
 
-- Append optional scope to _type_ if possible: `(lsp)`, `(treesitter)` or `(float/windows)`.
+- Append optional scope to _type_ if possible: `(lsp)`, `(treesitter)`, `(float)`...
+
+- The _description_ shouldn't start with a capital letter or end in a period (`.`) .
```
Comment (clason) [inline CONTRIBUTING.md:104]:
```suggestion
- The _description_ shouldn't start with a capital letter or end in a period (`.`).
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -89,17 +89,19 @@ the VCS/git logs more valuable. The general structure of a commit message is as
...
     - `perf`: performance improvements.
-    - `refactor`: modification of the code base which neither adds a feature nor fixes a bug - such as removing redundant code, simplifying the code, renaming variables, etc.
+    - `refactor`: modification of the code base which neither adds a feature nor fixes a bug - such as removing redundant code, simplifying code, renaming variables, etc.
     - `revert`: revert previous commits.
     - `test`: all changes related to tests such as refactoring existing tests or adding new tests.
     - `vim-patch`: all patches from upstream Vim. The commit messages for patches has [slightly different rules](https://github.com/neovim/neovim/wiki/Merging-patches-from-upstream-Vim#pull-requests) as not to interfere with existing scripts.
     - `chore`: Lastly, if none of the types above fits you may use `chore` as the type.
 
-- Append optional scope to _type_ if possible: `(lsp)`, `(treesitter)` or `(float/windows)`.
+- Append optional scope to _type_ if possible: `(lsp)`, `(treesitter)`, `(float)`...
```
Comment (clason) [inline CONTRIBUTING.md:102]:
```suggestion
- Append optional scope to _type_ such as `(lsp)`, `(treesitter)`, `(float)`, ...
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -89,17 +89,19 @@ the VCS/git logs more valuable. The general structure of a commit message is as
...
+
+- The _description_ shouldn't start with a capital letter or end in a period `.` .
```
Comment (clason) [inline CONTRIBUTING.md]:
```suggestion
- The _description_ shouldn't start with a capital letter or end in a period `.`.
```

**Review by clason — APPROVED** [review]

**Comment by dundargoc** [comment]
@clason whoa, that was a quick review.

**Comment by dundargoc** [comment]
@clason I apologize, I made a force push after making a pull request, I didn't except your lighting fast response times. Would you mind taking a look again?

**Review by clason — APPROVED** [review]
Yep, all good now! Sorry for jumping the gun.

**Pull request merged**